ckan/public/base/javascript/modules/resource-view-filters.js

```diff
@@ -103,26 +103,30 @@ this.ckan.module('resource-view-filters', function (jQuery) {
                 query;
 
             query = {
-              plain: false,
               resource_id: resourceId,
               limit: queryLimit,
               offset: offset,
               fields: filterName,
               distinct: true,
-              sort: filterName
+              sort: filterName,
+              include_total: false
             };
 
             if (term !== '') {
               var q = {};
-              q[filterName] = term + ':*';
+              if (!term.includes(' ')) {
```

Comment:
https://caniuse.com/#search=String.prototype.includes - should be replaced with `term.indexOf(' ') === -1` or(which i like) `!~term.indexOf(' ')`

Reply:
ah, I didn't see this and just went with `term.indexOf(' ') == -1` is there any significance to the `===` comparison for -1?